pom.xml

```diff
@@ -271,6 +271,13 @@
         <artifactId>google-http-client-gson</artifactId>
         <version>${google.http-client.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-storage-bom</artifactId>
+        <version>2.14.0</version>
```

Comment:
It should be `${google.cloud-storage.bom.version}` here.

Reply:
Mybad. Updated.